Neos.Neos/Classes/FrontendRouting/Projection/DocumentUriPathProjection.php

```diff
@@ -997,7 +997,8 @@ private function whenDimensionShineThroughWasAdded(DimensionShineThroughWasAdded
                     precedingnodeaggregateid,
                     succeedingnodeaggregateid,
                     shortcuttarget,
-                    nodetypename
+                    nodetypename,
+                    isplaceholder
```

Comment:
```suggestion
                    isplaceholder,
```

Reply:
Absolutely, yes

Reply:
Edit: reverted it, this is SQL, not PHP and not valid syntax